### PR TITLE
net-p2p/go-ethereum: package 1.14.8

### PR DESCRIPTION
**Pull request by SamWilsn**
go-ethereum released a new version.

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: net-p2p/go-ethereum/go-ethereum-1.14.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module
+
+LONG_VERSION="${PV}+build30152+noble"
+DESCRIPTION="Official golang implementation of the Ethereum protocol"
+HOMEPAGE="https://github.com/ethereum/go-ethereum"
+SRC_URI="https://ppa.launchpadcontent.net/ethereum/ethereum/ubuntu/pool/main/e/ethereum-unstable/ethereum-unstable_${LONG_VERSION}.tar.xz -> ${P}.tar.xz"
```
Comment (juippis) [inline net-p2p/go-ethereum/go-ethereum-1.14.8.ebuild:11]: I can't download the package from here.

Comment (SamWilsn) [reply net-p2p/go-ethereum/go-ethereum-1.14.8.ebuild]: I guess they stabilized 1.14.8 while I wasn't looking. Sorry about that!

Comment (juippis) [reply net-p2p/go-ethereum/go-ethereum-1.14.8.ebuild]: No need to be sorry, if we were faster handling this we would've gotten it mirrored with our mirror servers. Just unfortunate timing. Will check your updated PR in few hours!

File: net-p2p/go-ethereum/go-ethereum-1.14.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	# TODO: replace with wildcard
+	if use devtools; then
+		dobin build/bin/abidump
+		dobin build/bin/abigen
+		dobin build/bin/bootnode
+		dobin build/bin/clef
+		dobin build/bin/devp2p
+		dobin build/bin/ethkey
+		dobin build/bin/evm
+		dobin build/bin/p2psim
```
Comment (juippis) [inline net-p2p/go-ethereum/go-ethereum-1.14.8.ebuild:47]: With `+devtools` use flag this binary doesn't exist and the installation fails. Could you go through the updated file list? :)

**Pull request closed without merging**